fala/templates/adviser/scotland_warning.html

```diff
@@ -2,10 +2,9 @@
   <div class="govuk-warning-text" id="legal_aid_in_scotland_warning_text">
     <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
     <strong class="govuk-warning-text__text">
-      <span class="govuk-warning-text__assistive">Warning </span>
-      <p>These results cover England and Wales. Find out about
+      <span class="govuk-visually-hidden">Warning</span>
```

Comment:
Confirmed this is the correct action by checking [the changelog](https://github.com/alphagov/govuk-frontend/blob/30841046cccf5bb9ff963020a0bcf0287b11d5b6/CHANGELOG.md?plain=1#L831)